examples/rbac/rstudio-launcher-rbac-0.2.17.yaml

```diff
@@ -0,0 +1,88 @@
+---
```

Comment:
Should we be including a launcher rbac example in this PR?

IIRC these examples files are generated automatically from `charts/rstudio-launcher-rbac`

Reply:
That was generated by `just update-versions`.

Reply:
Sorry, confused repos _yet again_. 

> Note: The helm-docs job in CI takes care of updating the helm chart version in the README. See [example commit](https://github.com/rstudio/helm/pull/393/commits/cb8281b95a0ddd36534d772b0d4ae159bcb73c0a) from the actions-user bot.

Not sure why the bot edited more than the README?

Reply:
This is the workflow: https://github.com/rstudio/helm/actions/runs/5685442795/workflow#L65

Reply:
And the run that committed the rbac change: https://github.com/rstudio/helm/actions/runs/5685429289/job/15410307455

Reply:
oh! I missed that it was from the bot commit.

@colearendt It looks like [rstudio-launcher-rbac.yaml](https://github.com/rstudio/helm/blob/main/examples/rbac/rstudio-launcher-rbac.yaml) is no longer being kept up to date with the latest rbac example version (`0.2.17`), is that intentional?

Reply:
I think this was just a miss on a recent PR (i.e. we didn't regenerate b/c it was a PR from a fork). Should be fine to just merge it, I believe 😄